content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -33,9 +33,9 @@ This page describes how to use Admission Webhooks and Initializers.
 
 Admission webhooks are HTTP callbacks that receive admission requests and do
 something with them. You can define two types of admission webhooks,
-[validating admission Webhook](/docs/admin/admission-controllers/#validatingadmissionwebhook-alpha-in-18-beta-in-19)
+[validating admission Webhook](/docs/admin/admission-controllers/#validatingadmissionwebhook-alpha-in-1-8-beta-in-1-9)
```

Comment:
The link should be: docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook-beta-in-1-9

Reply:
I did some update based on your comments.
While this updated url seems not working properly with Chrome, the achor doesn't works.
Same url works fine in safari.

My original url: /docs/admin/admission-controllers/#validatingadmissionwebhook-alpha-in-1-8-beta-in-1-9
works for both browsers.

Reply:
https://k8s.io//docs/admin/admission-controllers/ redirects to 
https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/

So, let's use the new URL.